code/game/turfs/open/planet.dm

```diff
@@ -92,3 +92,7 @@
 
 /turf/closed/mineral/random/jungle/space_safe
 	baseturfs = /turf/open/misc/dirt/dark/station/airless
+
+//Monkestation Addition
+/turf/open/misc/grass/jungle/station // Same as the dirt please stop :(
```

Comment:
wouldn't the already existing `/turf/open/misc/grass/jungle/planet` work just fine? it's used for stuff at centcom.

Reply:
ah yep it would, didn't see that one changed it